go/vt/vtgate/planbuilder/testdata/dml_cases.json

```diff
@@ -4410,8 +4410,8 @@
     }
   },
   {
-    "comment": "explain dml without any directive should fail",
-    "query": "explain format=vtexplain delete from user",
+    "comment": "vexplain all dml without any directive should fail",
+    "query": "vexplain all delete from user",
     "plan": "VT09008: vexplain queries/all will actually run queries"
```

Comment:
It is not clear to me if `vtexplain all` or just `vtexplain` queries, will actually run queries

Reply:
both of them will. only `vexplain plan` does not run it.